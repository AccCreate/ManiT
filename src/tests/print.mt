print("Hello");
print(2);
print(2.2);
print(true);
print(false);
a = 2;
b = 2.2;
c = true;
d = false;
print(a);
print(b);
print(c);
print(d);