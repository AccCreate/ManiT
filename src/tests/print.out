Hello
2
2.2
true
false
Hello
2
2.2
true
false